src/tools/chip-cert/BUILD.gn

```diff
@@ -40,8 +33,6 @@ executable("chip-cert") {
     "chip-cert.h",
   ]
 
-  public_configs = [ ":openssl_config" ]
```

Comment:
Wouldn't it be enough to modify just this file and use "${chip_root}/crypto:openssl_config" here? I think it makes more sense to have `openssl_config` defined in `src/crypto/BUILD.gn` than in `BUILD.gn`.

Reply:
done